### ISSUE TITLE
Misleading comment in the code

### ISSUE THREAD

**Issue by mercmobily** [PolymerElements/iron-validator-behavior#22]
The code reads:

```
   /**
   * Name for this validator, used by `Polymer.IronValidatableBehavior` to lookup this element.
   */
  validatorName: {
    type: String,
    value: function() {
      return this.is;
    }
  }
```

The comment is misleading. In actual fact, `validatorName` is never _ever_ used by `Polymer.IronValidatableBehavior` which instead gets it from `<iron-meta>` directly:

```
 __computeValidator: function() {
   return Polymer.IronValidatableBehaviorMeta &&
      Polymer.IronValidatableBehaviorMeta.byKey(this.validator);
 }
```

The only spot `validatorName` is actually used is in `iron-validator-behavior` itself.

**Comment by notwaldorf** [comment]
Yeah, it should be deprecated (but that would be a breaking change, so we need to wait for a bit)

**Comment by mercmobily** [comment]
I would mark this in the documentation as deprecated now, so that people know? (As it's marked as such elsewhere...)

**Comment by mercmobily** [comment]
Sorry, just to clarify: my ticket wasn't about taking this out, but updating the comment/docs...

**Comment by notwaldorf** [comment]
Fixed this in the 2.0-preview branch
